pages/docs/reference/whatsnew14.md

```diff
@@ -574,3 +574,306 @@ Complete the initial configuration, and when you add a new dependency to `cocoap
 The new dependency will be added automatically. No additional steps are required.
 
 Learn [how to add dependencies](native/cocoapods.html).
+
+
+## Standard library
+
+### Extending the common library
```

Comment:
This section heading and the functions grouped by it look strange to me. As if the other sections on the same level are not about the common library.